app/dashboards/user_programme_enrolment_dashboard.rb

```diff
@@ -28,7 +28,7 @@ class UserProgrammeEnrolmentDashboard < Administrate::BaseDashboard
     user
     programme
     pathway
-    current_state
+    state_machine
```

Comment:
I'm not sure how I feel about calling it state_machine. The idea that its a state machine is an implementation detail. The admin user should be looking at the "current_state" which may or may not be powered by a state machine.